src/plugins/vis_builder/public/application/components/data_tab/utils/get_available_fields.test.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { FieldSpec } from '../../../../../../data/common';
+import { IndexPatternField, OSD_FIELD_TYPES } from '../../../../../../data/public';
+import { getAvailableFields } from './get_available_fields';
+
+describe('getAvailableFields', () => {
+  const createIndexFields = (fields: Array<Partial<FieldSpec>>) =>
+    fields.map(
+      (field) =>
+        new IndexPatternField(
+          {
+            aggregatable: true,
+            name: 'field 1',
```

Comment:
you could increment the name here based on the index - that way you wouldn't need to manually pass it

Reply:
Oh damm, didn't even realize that. Nice catch. Given that all the tests have already passed and I have your approval, you okay if i skip this since each test already passes the field name and neither the function nor the test actually relies on the name prop anyways.

Reply:
yeah, of course, not a blocker